src/providers/postgres/qgspostgresconn.h

```diff
@@ -211,13 +211,35 @@ class QgsPostgresConn : public QObject
     Q_OBJECT
 
   public:
-    /*
+
+    /**
+     * Get a new PostgreSQL connection
+     *
+     * \param connInfo the QgsDataSourceUri connection info with username / password
+     * \param readOnly is the connection read only ?
      * \param shared allow using a shared connection. Should never be
      *        called from a thread other than the main one.
      *        An assertion guards against such programmatic error.
+     * \param transaction is the connection a transaction ?
+     *
+     * \returns the PostgreSQL connection
      */
     static QgsPostgresConn *connectDb( const QString &connInfo, bool readOnly, bool shared = true, bool transaction = false );
```

Comment:
Is this still used anywhere? If not, you can safely remove it -- this class isn't stable API

Reply:
It is still used by QgsPostgresTransaction builder derived from QgsTransaction.